static/app/views/nav/secondary/sections/explore/exploreSecondaryNav.tsx

```diff
@@ -38,6 +38,15 @@ export function ExploreSecondaryNav() {
               {t('Logs')}
             </SecondaryNav.Item>
           </Feature>
+          <Feature features="discover-basic">
```

Comment:
Moved the discover item up below logs because we want it to be the default when traces/logs is not available.